### PR TITLE
Initial Charter Commit

### PR DESCRIPTION
**Pull request by Crypto2099**
Proposing to commit the initial  version of the Charter to the main readme file (until we have more documentation) and trying to set up a GitHub pages account for pretty formatting.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+As we move forward, we encourage open dialogue, continuous learning, and
+proactive engagement with the broader community. Through our combined efforts,
+we can ensure that the Cardano Atlantic Council remains a vital and effective
+component of Cardano's Constitutional Committee, championing the interests and
+principles of the entire Cardano ecosystem.
+
+## Signed and Accepted By:
+
+[Adam Dean](https://twitter.com/adamkdean)
```
Comment (rickymac68) [inline docs/README.md]:
```suggestion
[Adam Dean](https://twitter.com/adamkdean)
[Richard McCracken](https://x.com/RichardMcCrackn)
```

Comment (rickymac68) [inline docs/README.md]:
```suggestion
[Adam Dean](https://twitter.com/adamkdean)
[Richard McCracken](https://x.com/RichardMcCrackn)
```

**Review by rickymac68 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,298 @@
...
+
+As we move forward, we encourage open dialogue, continuous learning, and
+proactive engagement with the broader community. Through our combined efforts,
+we can ensure that the Cardano Atlantic Council remains a vital and effective
+component of Cardano's Constitutional Committee, championing the interests and
+principles of the entire Cardano ecosystem.
+
+## Signed and Accepted By:
+
+[Adam Dean](https://twitter.com/adamkdean)
```
Comment (Crypto2099) [inline README.md]:
```suggestion
[Adam Dean](https://twitter.com/adamkdean)
[Richard McCracken](https://x.com/RichardMcCrackn)
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,299 @@
...
+[Richard McCracken](https://x.com/RichardMcCrackn)
```
Comment (Hornan7) [inline README.md]:
```suggestion
[Richard McCracken](https://x.com/RichardMcCrackn)
[Mike Hornan](https://x.com/Hornan7)
```

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+[Richard McCracken](https://x.com/RichardMcCrackn)
```
Comment (Hornan7) [inline docs/README.md]:
```suggestion
[Richard McCracken](https://x.com/RichardMcCrackn)
[Mike Hornan](https://x.com/Hornan7)
```

Comment (thiago-tsnnst) [inline docs/README.md]: [Thiago Nunes](https://x.com/tsnnst)

File: README.md [hunk 5]
```diff
@@ -1,2 +1,299 @@
...
+
+As we move forward, we encourage open dialogue, continuous learning, and
+proactive engagement with the broader community. Through our combined efforts,
+we can ensure that the Cardano Atlantic Council remains a vital and effective
+component of Cardano's Constitutional Committee, championing the interests and
+principles of the entire Cardano ecosystem.
+
+## Signed and Accepted By:
+
+[Adam Dean](https://twitter.com/adamkdean)
```
Comment (thiago-tsnnst) [inline README.md]: [Adam Dean](https://twitter.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,300 @@
...
+[Richard McCracken](https://x.com/RichardMcCrackn)
+[Mike Hornan](https://x.com/Hornan7)
```
Comment (Crypto2099) [inline README.md]:
```suggestion
## Signed and Accepted By:

[Adam Dean](https://x.com/adamkdean)
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)
[Mike Hornan](https://x.com/Hornan7)
[Richard McCracken](https://x.com/RichardMcCrackn)
[Thiago Nunes](https://x.com/tsnnst)
```

File: docs/README.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+[Mike Hornan](https://x.com/Hornan7)
```
Comment (Crypto2099) [inline docs/README.md]:
```suggestion
## Signed and Accepted By:

[Adam Dean](https://x.com/adamkdean)
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)
[Mike Hornan](https://x.com/Hornan7)
[Richard McCracken](https://x.com/RichardMcCrackn)
[Thiago Nunes](https://x.com/tsnnst)
```

**Review by Crypto2099 — COMMENTED** [review]
Final changes to add the rest of the signers after verbal confirmation of agreement

**Review by Hornan7 — APPROVED** [review]

**Pull request merged**